packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/guides/Migration.tool.md

```diff
@@ -0,0 +1,522 @@
++ const webpackConfig = require('@cerner/webpack-config-terra');
+```
+
+Update any references to the old terra-aggregate-translations:
+
+
+```diff
+// jestGlobalSetup.js
+- const aggregateTranslations = require('terra-aggregate-translations');
++ const aggregateTranslations = require('@cerner/terra-aggregate-translations');
+```
+
+Update any scripts in the package.json to use webpack-dev-server. Use terra-cli for any static sites. Remove `terra-aggregate-translations` if installed:
+
+```diff
+// package.json
+{
+  "scripts": {
+-    "start": "tt-serve",
+-    "start-prod": "tt-serve --env.disableHotReloading -p",
+-    "start-static": "npm run pack && tt-serve-static",
++    "start": "webpack-dev-server",
++    "start-prod": "webpack-dev-server --env.disableHotReloading -p",
++    "start-static": "npm run pack && terra express-server --site ./build",
+  },
+  "devDependencies": {
++   "@cerner/terra-aggregate-translations": "^2.0.1",
++   "@cerner/terra-cli": "^1.3.0",
++   "@cerner/webpack-config-terra": "^1.2.0",
++   "postcss": "^8.2.1",
+-   "terra-aggregate-translations": "^1.0.0"
+  }
+}
+```
+
+At this point it is recommended to do a clean install prior to testing each of the scripts. If you have the script go ahead and run `npm run clean:install`. Alternatively ensure that the old node_modules directory is deleted and reinstall all dependencies.
+
+```sh
+npm run clean:install
+```
+
+Verify each of the scripts are working as intended:
+
+```sh
+npm run start
+
+npm run start-prod
+
+npm run start-static
+```
+
+Note: If you don't have some of these commands it is not necessary to add them. Just ensure the commands you do have are working as expected. All of the `tt-serve` commands should be updated to use webpack-dev-server or the terra express server
+
+### Upgrading ESlint Config Terra
+
+If you are already using @cerner/eslint-config-terra ^4.0.0 you can skip this section.
+
+Upgrading `eslint-config-terra` is optional, but recommended. WebDriverIO v6 syntax requires using the `$` global which has been [added](https://github.com/cerner/terra-toolkit/pull/548) to the eslint config. If you choose not to upgrade eslint-config-terra at this time you will need to add the global declaration manually in each spec file.
+
+Adding the global manually to each spec file is only necessary if you are not upgrading to @cerner/eslint-config-terra ^4.0.0:
+
+```js
+/* global $ */
+```
+
+A complete upgrade guide can be found [here](https://github.com/cerner/terra-toolkit/blob/main/packages/eslint-config-terra/UpgradeGuide.md#eslint-config-terra-upgrade-guide).
+
+Install @cerner/eslint-config-terra:
+
+```sh
+npm install --save-dev @cerner/eslint-config-terra
+```
+
+Remove the old `eslint-config-terra` and extend `@cerner/terra` from the `eslintConfig` key:
+
+```diff
+// package.json
+{
+  "eslintConfig": {
+-   "extends": "terra",
++   "extends": "@cerner/terra",
+  },
+  "devDependencies": {
+-    "eslint-config-terra": "^3.3.0",
```

Comment:
```suggestion
-    "eslint-config-terra": "^3.3.0",
-    "eslint": "^6.0.0"
```

Reply:
Added ^7.0.0 from the previous comment suggestion:

https://github.com/cerner/terra-toolkit/pull/574/commits/42750ce4526f2f3d447820f56d3dd5b0f3b8bfc2